src/commands/quote.ts

```diff
@@ -90,13 +90,18 @@ async function parseQuotes(client: Client, str: string) {
     }
   )
 
+
   const parsed_emotes = await extractObjects(
     parsed_mentions,
-    /(?<=<:(.+?))\d+(?=>)/g,
-    /<:(.+?):\d+>/g,
+    /(?<=<:)(.+?):\d+>/g,
+    /<:.+?:\d+>/g,
     async (extracted) => {
-      // placeholder for the replacement algorithm for emojis
-      return ""
+      const emoji = client.emojis.resolve(extracted)
```

Comment:
Why is this here..? You're getting the emojis and returning nothing if an emote is there?

Have you tested this?